### PR TITLE
Fix additional attributes when using local scope

### PR DESCRIPTION
**Pull request by arxeiss**
Fix error: `Property [additional_attributes] does not exist on the Eloquent builder instance.`

This happens, when a local scope is used. Then `$model` does not contain Model but Builder.

### CODE REVIEW

File: src/Http/Controllers/VoyagerBaseController.php [hunk 1]
```diff
@@ -868,7 +870,7 @@ public function relation(Request $request)
                 // If search query, use LIKE to filter results depending on field label
                 if ($search) {
                     // If we are using additional_attribute as label
-                    if (in_array($options->label, $model->additional_attributes ?? [])) {
```
Comment (MrCrayon) [inline src/Http/Controllers/VoyagerBaseController.php:871]: I don't understand, you just moved this.
How is it changing anything?

Comment (emptynick) [reply src/Http/Controllers/VoyagerBaseController.php:871]: The scope is applied in between his changes.
$model changes it's type then

Comment (arxeiss) [reply src/Http/Controllers/VoyagerBaseController.php:871]: When you apply local scope by this `$model = $model->{$options->scope}();` then `$model` does not contains model instance but Builder instance which then does not support `additional_attributes`.

Try to create relation in BREAD with local scope. When you try to select it, it works but when you try to search/filter, it doesn't work. See my MR description, there is original message which is thrown

**Review by MrCrayon — COMMENTED** [review]
I see thanks

**Comment by emptynick** [comment]
I'm more curious if the accessor is available on a `Builder` instance then?

**Comment by moisesh18** [comment]
This worked for me. Why it is not in merged?

**Review by moisesh18 — APPROVED** [review]

**Review by emptynick — APPROVED** [review]

**Pull request merged**